src/modules/emails/dto/post-emails-request.dto.ts

```diff
@@ -58,7 +57,8 @@ export class PostEmailsRequestDto {
   @MaxLength(400)
   @ApiProperty({
     example: GOVUK_NOTIFY.EXAMPLES.FILE,
-    description: 'File for GovNotify to consume and generate a link to download',
+    description:
+      'File for GovNotify to consume and generate a link to download with supported file types CSV (.csv), image (.jpeg, .jpg, .png), Microsoft Excel Spreadsheet (.xlsx), Microsoft Word Document (.doc, .docx), PDF (.pdf), text (.json, .odt, .rtf, .txt). File size mus tbe smaller than 2MB.',
```

Comment:
I suggest, simply e.g:

```ts
File for GovNotify to consume and generate a link to download with supported file types. 
The file size must be smaller than ${NEW_CONSTANT}.
```

Otherwise, the file types could change and then this documentation would be out of date, same with the file size

Reply:
Otherwise without the suggestion, there is a typo 😛 `file size mus tbe smaller`